Copyedit investigar literatura lusofona literateca

joanacvp opened the pull request:
Boa tarde @dianamsmpsantos. 
Muito obrigada pela lição! Foi óptimo retomar o contacto com a Literateca. Aprendi muito e acho que está muito bem conseguida.
Fiz pequenos ajustes de copyedit, alguns dos quais tendo em conta o nosso glossário de compatibilidade. Também actualizei o link da sua última lição sobre R (que estava quebrado). Nas notas de fim substituí alguns links por DOI, quando existente, pois costuma ser o utilizado no PH.
Tenho apenas uma questão em relação à Figura 2. No final da mesma, no localizador da página web, aparece a pesquisa "par de botas". Esta será para constar da imagem? (Fiquei na dúvida porque não é um termo mencionado no texto).

Fica em falta as secções activity, topics, abstract e avatar_alt da secção inicial, que creio que serão pedidas na fase seguinte

Muito obrigada!

joanacvp committed:
Update investigar-literatura-lusofona-literateca.md

Will get back to it. Left at line 159.

joanacvp committed:
Update investigar-literatura-lusofona-literateca.md

Left on line 237

joanacvp committed:
Update investigar-literatura-lusofona-literateca.md

Finished the revision. Will come back for a final reading.

joanacvp committed:
Update investigar-literatura-lusofona-literateca.md

Copy Edit finished.

joanacvp commented on pt/esbocos/originais/investigar-literatura-lusofona-literateca.md:
```diff
@@ -234,7 +233,7 @@ Relembrando que
 * escolhemos o corpo Literateca
 * escolhemos a opção tsv
 
-e temos de guardar os ficheiros com nomes descritivos. No caso em questão, chamei-lhes `distribuicaoAparenciaDecadaGen.tsv` [distribuicaoAparenciaDecadaGen.tsv](https://github.com/programminghistorian/ph-submissions/blob/gh-pages/assets/investigar-literatura-lusofona-literateca/distribuicaoAparenciaDecadaGen.tsv) e `distribuicaoPredDecadaGen.tsv` [distribuicaoPredDecadaGen.tsv](https://github.com ...
```
Olá @dianamsmpsantos. Uma vez que todos os ficheiros tsv gerados têm linhagem para o GitHub, questiono se, no próprio nome, não poderia estar logo o link (em vez de repetir o mesmo). Isto acontece aqui, mas também noutras zonas da lição. Obrigada

dianamsmpsantos replied:
SIm, penso que isso ficaria muiro melhor. Mudo eu, ou mudam vocês?

joanacvp replied:
Olá @dianamsmpsantos. Já fiz essa mudança.

dianamsmpsantos commented:
Muito obrigada, Joana! Aqui estão alguns pequenos casos em que eu não
concordo com a revisão:

51 chama de metadados -> chama metadados
74 ao que se dá -> a que se dá
85 , bem como -> . Tanto permite identificar em contexto o resultado (para
leitura próxima) como produzir um resumo quantitativo (a chamada leitura
distante).
144 e o nome da variante -> e a quarta o nome da variante
179 visualizá-lo -> visualizar o resultado
194 verificar -> considerar
202 vai aumenta -> vai aumentando
278 visualizá-lo -> visualizar essa questão
294 locais relativos -> locxais relativamente
315 anotámos OBras -> anotámos o corpo OBras

Um abraço
Diana
P.S, Em relação ao "par de botas", vou enviar à Charlotte outra figura que
não tenha o dito! Muito obrigada pela deteção desse caso, que eu não faço
ideia donde veio :)

joanacvp ***@***.***> escreveu (terça, 8/04/2025 à(s) 18:21):

> ***@***.**** commented on this pull request.
> ------------------------------
>
> In pt/esbocos/originais/investigar-literatura-lusofona-literateca.md
> <https://github.com/programminghistorian/ph-submissions/pull/668#discussion_r2033572584>
> :
>
> > @@ -234,7 +233,7 @@ Relembrando que
>  * escolhemos o corpo Literateca
>  * escolhemos a opção tsv
>
> -e temos de guardar os ficheiros com nomes descritivos. No caso em questão, chamei-lhes `distribuicaoAparenciaDecadaGen.tsv` [distribuicaoAparenciaDecadaGen.tsv](https://github.com/programminghistorian/ph-submissions/blob/gh-pages/assets/investigar-literatura-lusofona-literateca/distribuicaoAparenciaDecadaGen.tsv) e `distribuicaoPredDecadaGen.tsv` [distribuicaoPredDecadaGen.tsv](https://github.com/programminghistorian/ph-submissions/blob/gh-pages/assets/investigar-literatura-lusofona-literateca/distribuicaoPredDecadaGen.tsv).
>
> Olá @dianamsmpsantos <https://github.com/dianamsmpsantos>. Uma vez que
> todos os ficheiros tsv gerados têm linhagem para o GitHub, questiono se, no
> próprio nome, não poderia estar logo o link (em vez de repetir o mesmo).
> Isto acontece aqui, mas também noutras zonas da lição. Obrigada
>
> —
> Reply to this email directly, view it on GitHub
> <https://github.com/programminghistorian/ph-submissions/pull/668#pullrequestreview-2750622036>,
> or unsubscribe
> <https://github.com/notifications/unsubscribe-auth/AB6H44US2LXJALFI3KOZUSL2YPZRLAVCNFSM6AAAAAB2WV7UB6VHI2DSMVQWIX3LMV43YUDVNRWFEZLROVSXG5CSMV3GSZLXHMZDONJQGYZDEMBTGY>
> .
> You are receiving this because you were mentioned.Message ID:
> ***@***.***
> com>
>

charlottejmc committed:
Replace Figure 2

charlottejmc commented:
Hello @dianamsmpsantos and @joanacvp, I've replaced Figure 2 (in this branch) with the new version, which Diana sent me by email.

joanacvp committed:
Update investigar-literatura-lusofona-literateca.md

Introduced the changes agreed with the author. The lesson is ready for next stage.

joanacvp commented:
Hello. I just implemented all the changes (and also the change on the reference and links to the tsv files). If @dianamsmpsantos agrees, I think it is ready to move to the next stage 🥇

dianamsmpsantos commented:
I was not able to check -- where should I look? But if everything is done,
of course I agree to move to the next stage :)
Diana

joanacvp ***@***.***> escreveu (quinta, 10/04/2025 à(s)
13:46):

> Hello. I just implemented all the changes (and also the change on the
> reference and links to the tsv files). If @dianamsmpsantos
> <https://github.com/dianamsmpsantos> agrees, I think it is ready to move
> to the next stage 🥇
>
> —
> Reply to this email directly, view it on GitHub
> <https://github.com/programminghistorian/ph-submissions/pull/668#issuecomment-2792475336>,
> or unsubscribe
> <https://github.com/notifications/unsubscribe-auth/AB6H44QIE4LX6TM3KDN6DS32YZK3DAVCNFSM6AAAAAB2WV7UB6VHI2DSMVQWIX3LMV43OSLTON2WKQ3PNVWWK3TUHMZDOOJSGQ3TKMZTGY>
> .
> You are receiving this because you were mentioned.Message ID:
> ***@***.***>
> *joanacvp* left a comment (programminghistorian/ph-submissions#668)
> <https://github.com/programminghistorian/ph-submissions/pull/668#issuecomment-2792475336>
>
> Hello. I just implemented all the changes (and also the change on the
> reference and links to the tsv files). If @dianamsmpsantos
> <https://github.com/dianamsmpsantos> agrees, I think it is ready to move
> to the next stage 🥇
>
> —
> Reply to this email directly, view it on GitHub
> <https://github.com/programminghistorian/ph-submissions/pull/668#issuecomment-2792475336>,
> or unsubscribe
> <https://github.com/notifications/unsubscribe-auth/AB6H44QIE4LX6TM3KDN6DS32YZK3DAVCNFSM6AAAAAB2WV7UB6VHI2DSMVQWIX3LMV43OSLTON2WKQ3PNVWWK3TUHMZDOOJSGQ3TKMZTGY>
> .
> You are receiving this because you were mentioned.Message ID:
> ***@***.***>
>

joanacvp commented:
Hi @dianamsmpsantos. You can check here: [https://github.com/programminghistorian/ph-submissions/blob/copyedit-investigar-literatura-lusofona-literateca/pt/esbocos/originais/investigar-literatura-lusofona-literateca.md](https://github.com/programminghistorian/ph-submissions/blob/copyedit-investigar-literatura-lusofona-literateca/pt/esbocos/originais/investigar-literatura-lusofona-literateca.md) (the copyediting branch).

dianamsmpsantos commented:
Everything is perfect, thanks a lot!
Diana

joanacvp ***@***.***> escreveu (quinta, 10/04/2025 à(s)
16:14):

> Hi @dianamsmpsantos <https://github.com/dianamsmpsantos>. You can check
> here:
> https://github.com/programminghistorian/ph-submissions/blob/copyedit-investigar-literatura-lusofona-literateca/pt/esbocos/originais/investigar-literatura-lusofona-literateca.md
> (the copyediting branch).
>
> —
> Reply to this email directly, view it on GitHub
> <https://github.com/programminghistorian/ph-submissions/pull/668#issuecomment-2793574685>,
> or unsubscribe
> <https://github.com/notifications/unsubscribe-auth/AB6H44VKNTHVKUVM54KVFM32YZ4E5AVCNFSM6AAAAAB2WV7UB6VHI2DSMVQWIX3LMV43OSLTON2WKQ3PNVWWK3TUHMZDOOJTGU3TINRYGU>
> .
> You are receiving this because you were mentioned.Message ID:
> ***@***.***>
> *joanacvp* left a comment (programminghistorian/ph-submissions#668)
> <https://github.com/programminghistorian/ph-submissions/pull/668#issuecomment-2793574685>
>
> Hi @dianamsmpsantos <https://github.com/dianamsmpsantos>. You can check
> here:
> https://github.com/programminghistorian/ph-submissions/blob/copyedit-investigar-literatura-lusofona-literateca/pt/esbocos/originais/investigar-literatura-lusofona-literateca.md
> (the copyediting branch).
>
> —
> Reply to this email directly, view it on GitHub
> <https://github.com/programminghistorian/ph-submissions/pull/668#issuecomment-2793574685>,
> or unsubscribe
> <https://github.com/notifications/unsubscribe-auth/AB6H44VKNTHVKUVM54KVFM32YZ4E5AVCNFSM6AAAAAB2WV7UB6VHI2DSMVQWIX3LMV43OSLTON2WKQ3PNVWWK3TUHMZDOOJTGU3TINRYGU>
> .
> You are receiving this because you were mentioned.Message ID:
> ***@***.***>
>

joanacvp commented:
Perfect! @charlottejmc the copy edit is finished. Should I merge the pull request or should you?

charlottejmc commented:
Great! I'll merge the copyedits now, and then take the lesson into my own hands for typesetting and other final Phase 6 checks. Thank you all ✨
